### PR TITLE
Remove Cairo font family reference globally

### PR DESCRIPTION
**Pull request by julioest**
This PR removes the Cairo font family and sets a modern fallback sans-serif font stack

This addresses #1065

### CODE REVIEW

**Comment by julioest** [comment]
@vinniefalco Here's the PR disabling the Cairo font for now

**Comment by julioest** [comment]
**Previews:** 
boost-releases
boost-libraries
boost-community
boost-learn
boost-home

**Comment by kennethreitz** [comment]
@julioest based on the comments in #1065, do you think we should merge this?

**Comment by julioest** [comment]
@kennethreitz Thanks for the update, got it now. It's not ready to merge; I'm going to remove all Cairo references first

**Comment by kennethreitz** [comment]
Sounds good!

**Comment by julioest** [comment]
Cairo removed

File: static/css/styles.css [hunk 1]
```diff
@@ -2637,6 +2629,10 @@ code,
   padding-left: 1rem;
 }
 
+.pb-0 {
+  padding-bottom: 0px;
+}
+
```
Comment (vinniefalco) [inline static/css/styles.css:2635]: Why are there all these small diffs?

Comment (julioest) [reply static/css/styles.css:2635]: This is tailwind purging, adding, and organizing classes used/not-used in the markup

Comment (vinniefalco) [reply static/css/styles.css:2635]: oh, this file is generated by tailwind?

Comment (julioest) [reply static/css/styles.css:2635]: Yes, correct!

**Pull request merged**